include/misc/timing_info.h

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
The syscalls defined here are specific to a single test case.
This should *not* go in include/misc/, this needs to be in the same codebase as the test case.
We might need to extend the syscall auto-magic to have additional search paths,